.ci/magic-modules/coverage-spreadsheet-upload.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Service account credentials for GCP to allow terraform to work
+export GOOGLE_CLOUD_KEYFILE_JSON="/tmp/google-account.json"
+export GOOGLE_APPLICATION_CREDENTIALS="/tmp/google-account.json"
+
+# CI sets the contents of our json account secret in our environment; dump it
+# to disk for use in tests.
+set +x
+echo "${SERVICE_ACCOUNT}" > /tmp/google-account.json
```

Comment:
How isolated are our builds? i.e., can a subsequent build come along and read this file?

Reply:
Every build is destroyed after finished. In my experience, that process seems to happen within ~30s of the build finishing. Builds are isolated in Docker containers, so I'm not too concerned about data leaks between builds.